packages/assistify-ai/server/methods/SmartiReload.js

```diff
@@ -137,8 +117,17 @@ Meteor.methods({
 			conversation.messages.push(newMessage);
 		}
 
-		SmartiProxy.propagateToSmarti(verbs.post, 'conversation', conversation);
-		SystemLogger.debug('New conversation updated with Smarti');
+		// post the new conversation to Smarti
+		const newSmartiConversation = SmartiProxy.propagateToSmarti(verbs.post, 'conversation', conversation);
+		// get the analysis result
+		const analysisResult = SmartiProxy.propagateToSmarti(verbs.get, `conversation/${ newSmartiConversation.id }/analysis`);
```

Comment:
When submitting `analysis: true` in the body of the post, the analysis is synchronously returned. Thus, the get can be omitted.